recipes/openimageio/all/conanfile.py

```diff
@@ -75,11 +77,14 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+        if Version(self.version) < "3.0.0.0":
+            self.options.with_libjxl = False  #rm_safe("with_libjxl")
```

Comment:
Option values can not be changed after the `config_options` method, `configure()` is too late

- If this is a default chaning with the versions, move this check to `config_options`
- If the recipe no longer has this option, remove it as in the comment
- If it supports the option, but for some reason True is failing to compile right now, adding a check in the validate() methiod would be preferred :)

Reply:
Hey @AbrilRBS, thanks for following up. I had recently tried to rework the whole PR a bit but haven't got it to fully link. Just merged the state frem here and into my local state. I now followed your earlier suggestion and split it by creating a 3.x folder for the new stuff. Wasn't sure if I should rename the existing stuff probably?
(openexr has a 2.x and 3.x folder)

(Still pushed to have it up here, but it will likely fail to link. Up for suggestions as I've only had minimal time but also didn't got why that could now fail.